src/audio/dai-legacy.c

```diff
@@ -40,10 +40,10 @@ static const struct comp_driver comp_dai;
 LOG_MODULE_REGISTER(dai_comp, CONFIG_SOF_LOG_LEVEL);
 
 /* c2b00d27-ffbc-4150-a51a-245c79c5e54b */
-SOF_DEFINE_UUID("dai_legacy", dai_legacy_uuid, 0xc2b00d27, 0xffbc, 0x4150,
+SOF_DEFINE_UUID("dai", dai_uuid, 0xc2b00d27, 0xffbc, 0x4150,
 		 0xa5, 0x1a, 0x24, 0x5c, 0x79, 0xc5, 0xe5, 0x4b);
 
-DECLARE_TR_CTX(dai_comp_tr, SOF_UUID(dai_legacy_uuid), LOG_LEVEL_INFO);
+DECLARE_TR_CTX(dai_comp_tr, SOF_UUID(dai_uuid), LOG_LEVEL_INFO);
```

Comment:
is this part of the unification needed for some automation in one of the following commits? If not - maybe we could keep different names, makes grepping the code easier

Reply:
Actually this is sort of a fakeout.  The names in-tree are both "dai", and they were renamed by an early patch in this series to disambiguate them.  But then it turns out that the **value** of the UUID is the same in both cases, which is unfortunate but technically unresolvable (I mean... not really, but I'm trying very hard not to have to renumber these things and being really clear where I do).  So this moves back to the original usage where the two drivers share the same UUID, which is OK in this case as they can't be built into the same image anyway.

I should fold them together to reduce the churn, though that's a mildly hard rebase.